### ISSUE TITLE
[Feature Request | Installer] Option to overwrite current OptiFine profile in the minecraft launcher or create new profile

### ISSUE THREAD

**Issue by ilm9001** [sp614x/optifine#5170]
## The OptiFine installer should have a option to overwrite the current OF profile or create a new one with the newly installed version.

The current behavior is that optifine ALWAYS overwrites the existing minecraft installer optifine profile, if optifine has already been installed. So if you were to install optifine for minecraft version 1.16.4, then install it for 1.8.9 the optifine profile would only consist of optifine for 1.8.9.

My suggestion is that there should be a option to either overwrite or create a new profile in the launcher, so you can easily create a new OF profile which would be especially useful for major version changes like 1.15.2 -> 1.16.4. If you choose overwrite, it should let you choose which profile to overwrite.
![image](https://user-images.githubusercontent.com/54047826/103447413-b2498c00-4c93-11eb-8276-0b4de5635b93.png)

**Comment by Cart3r1234** [comment]
I think it should always overwrite for minor versions (like 1.16.3 -> 1.16.4), but the option to prevent it from overwriting major versions would be nice.

**Comment by ilm9001** [comment]
> I think it should always overwrite for minor versions (like 1.16.3 -> 1.16.4), but the option to prevent it from overwriting major versions would be nice.

While this was my original suggestion, a toggle in the launcher would be better. Some people just might want to have a different profile even for minor version changes.

**Comment by Cart3r1234** [comment]
True.  I think it would be best if there was a button/toggle in the optifine installer, rather than something to do with the launcher.  or make it two buttons, so instead of having just "install," have "install and overwrite" and "install to new profile" or something along those lines.

**Comment by ilm9001** [comment]
> True. I think it would be best if there was a button/toggle in the optifine installer
That's what my suggestion is. I edited the original post to clarify that a bit more.
